pkg/gh.go

```diff
@@ -15,5 +16,14 @@ func GithubClient() *github.Client {
 		httpclient = oauth2.NewClient(context.Background(), oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token}))
 	}
 	githubClient := github.NewClient(httpclient)
+	rateLimit, _, err := githubClient.RateLimits(context.Background())
```

Comment:
This is not resolved yet, the `githubClient.RateLimits(context.Background())` should be removed, otherwise it might consume 1 request each time the client is created.